src/utils.cpp

```diff
@@ -21,7 +33,7 @@ void Matrix::add(double n)
     {
         for (size_t j = 0; j < this->cols; j++)
         {
-            this->matrix[i][j] += n;
+            this->m[i][j] += n;
```

Comment:
As mentioned in the header file, all occurrences of this->m should be replaced by this->matrix